app/views/placements/support/organisations/users/_check_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%# locals: (form_url: "", cancel_url: "", user: nil, organisation: nil) -%>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= form_with(model: user, url: form_url, method: :post) do |f| %>
+        <%= f.hidden_field :first_name, value: @user.first_name %>
+        <%= f.hidden_field :last_name, value: @user.last_name %>
+        <%= f.hidden_field :email, value: @user.email %>
+
+        <label class="govuk-label govuk-label--l">
+          <span class="govuk-caption-l"><%= t(".caption", organisation_name: organisation.name) %></span>
+          <%= t(".title") %>
+        </label>
+
+        <dl class="govuk-summary-list">
```

Comment:
Is there a reason we are not using the govuk_summary_list helper? I think it's a bit neater.

Reply:
resolved in [853411c](https://github.com/DFE-Digital/itt-mentor-services/pull/125/commits/853411c3dc199496f425ee6adeb80fc5690ef163)